Sprint-3/alarmclock/style.css

```diff
@@ -13,3 +13,35 @@
 h1 {
   text-align: center;
 }
+.quadrat {
+  width: 100%;
+  height: 1000px;
+  background-color: blue;
+  color: white;
+  text-align: center;
+  line-height: 100px;
+  cursor: pointer;
+  transition: all 0.5s ease; /* Smooth transition for changes */
+}
+
+.disabled {
+  background-color: gray;
+  pointer-events: none;
+}
+
+/* Flashing effect */
+@keyframes flash {
```

Comment:
Nice to see you have explored some css effects

Reply:
thank you. i just reading codes from codepen it's easy to undrstand some of them